requirements.txt

```diff
@@ -1,4 +1,4 @@
-python-telegram-bot==20.1
+python-telegram-bot>=20.1
```

Comment:
Why this change ?

Reply:
The use of `>=` here is to make sure we use the newest telegram-bot package, I think it's necessary to keep the tg bot up-to-date for bug fix

Reply:
I am really unsure about this, but I think that handling bump manually is a better practice to avoid unexpected breaking changes. 
Although I am not really fearful of breaking changes happening with this lib, so it could be applied, indeed. Seems ok to me.

Reply:
From [PEP440](https://peps.python.org/pep-0440/#compatible-release), the `>=20.1` is a equivalent of `==20.*` which will match the latest `20.x` version of bot; It's unlikely to have a breaking changes on the minor version from my experience.